lib/shopify_app/sessions_controller.rb

```diff
@@ -35,7 +35,7 @@ def destroy
 
     def authenticate
       if shop_name = sanitize_shop_param(params)
-        fullpage_redirect_to "/auth/shopify?shop=#{shop_name}"
+        fullpage_redirect_to "#{main_app.root_path}auth/shopify?shop=#{shop_name}"
```

Comment:
I guess it's safe to rely on `root_path` always being there? In a vanilla rails app it isn't, but the first thing you're told to do is set it in `config/routes.rb`.